code/modules/projectiles/ammunition/energy.dm

```diff
@@ -75,6 +75,9 @@
 	fire_sound = 'sound/weapons/taser.ogg'
 	e_cost = 200
 
+/obj/item/ammo_casing/energy/electrode/hos
+	e_cost = 200

```

Comment:
Difference is that the other 200 cost electrode uses a difference sound when firing.